### ISSUE TITLE
Newsstand hangs on "buying" issue & nothing happens. Quit & go back and Newsstand says item is purchased, but users cannot download the issue

### ISSUE THREAD

**Issue by knoxbronson** [bakerframework/baker#499]
This happened to my landlady, to a friend, and now to the publisher of the magazine. 
My landlady bought the issue on the Newsstand (she had already downloaded free issues with no problem). The Newsstand said "Buying" and the little activity icon spun around forever. I quit out of the app, went back to buy again and it said "This in app purchase has already been bought. It will be restored for free." It took forever. See screenshots below.
I don't know what finally happened, but at some point the download indicator appeared and we were able to open the issue with no further problems. A friend in SoCal had a simillar problem and was not able to subscribe at all: it just timed out. Now the publisher of the magazine is having the same problem and I am catching hell. What is going on here? Is it our hosting platform, KnownHost, timing out or something? Is it a slow upload speed on our local wifi?
I had no problem subscribing a couple months ago. Not everyone is having this problem. We are getting some subscriptions and single issue purchases, but I fear that this has happened more than I know.
After all the other issues we've encountered putting out this magazine, this is frustrating beyond words.
Please help. 
Thank you.
![img_0788](https://cloud.githubusercontent.com/assets/7762179/9321674/86346d9c-451e-11e5-80f7-75784fef74cd.PNG)
![img_0790](https://cloud.githubusercontent.com/assets/7762179/9321673/86333210-451e-11e5-99bf-18b1958d45a0.PNG)

**Comment by nin9creative** [comment]
Hey Knox, replied to your email.  See if that fixes/resolves the issue.

**Comment by knoxbronson** [comment]
Thanks Andrew. Must wait for Gustavo. Crossing fingers!

Knox


Knox Bronson
P1xels—The Art of the iPhone
http://pixelsatanexhibition.com
510-612-6124

**Comment by hecklerguy** [comment]
i am having the same issue. How do i fix?

**Comment by MagLoftMagazines** [comment]
Hi guys, did you also get a chance to look at this bug that was resolved some time ago: https://github.com/bakerframework/baker/issues/293 ? Not sure if that's what's happening now, but it sounds similar. I know this was also added to master some time ago, and if you are using an older version of Baker, then perhaps the fix isn't in there.

Unfortunately it means you need to re-submit the app and risk it being rejected the first time due to an alert box that shows up. Maybe you can trigger this by going through Test Flight app first? Hope it helps.

**Comment by hecklerguy** [comment]
I'm using baker 4.3

**Comment by knoxbronson** [comment]
We had a naming issue which Andrew detected. When we made sure the issue had the same name everywhere, the problem went away.

Knoc

Knox Bronson
P1xels—The Art of the iPhone
http://pixelsatanexhibition.com
510-612-6124

**Comment by hecklerguy** [comment]
So do you mean all the issues titles in the iTunes connect, my baker cloud titles and the self.json?

**Comment by knoxbronson** [comment]
No, we had just one issue that was a problem, the first paid one. The first two free issues were fine.

This is what Andrew wrote and did and it seemed to fix it (am including a screenshot, don’t know if it will upload to the server via this email):

One thing I noticed that was weird is that some ProductID's are registered as issue03 and some are july2015.  Did someone set up Baker Cloud with issue03 and then later change it to july2015?

I deleted those rows because there were duplicate User ID's with both issue03 and july2015 showing...  Maybe that was the issue?





Knox Bronson
P1xels—The Art of the iPhone
http://pixelsatanexhibition.com
510-612-6124
